addons/test_mail/tests/test_mail_activity.py

```diff
@@ -89,6 +89,22 @@ def _employee_crash(*args, **kwargs):
 @tests.tagged('mail_activity')
 class TestActivityFlow(TestActivityCommon):
 
+    @classmethod
+    def setUpClass(cls):
+        super(TestActivityFlow, cls).setUpClass()
```

Comment:
I would move all tests in TestActivityMixin as those tests are related to internals of the mixin itself.